<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="1" skipped="0" tests="4" time="0.079" timestamp="2022-12-19T17:44:45.146967" hostname="BAN-LAP-YEJJALA"><testcase classname="test_multiplication" name="test_mutiplicaion_11[1-11]" time="0.001" /><testcase classname="test_multiplication" name="test_mutiplicaion_11[2-20]" time="0.001"><failure message="assert (11 * 2) == 20">num = 2, output = 20

    @pytest.mark.parametrize("num, output", [(1,11), (2,20), (3,33), (4,44)])
    def test_mutiplicaion_11(num, output):
&gt;       assert 11*num == output
E       assert (11 * 2) == 20

test_multiplication.py:5: AssertionError</failure></testcase><testcase classname="test_multiplication" name="test_mutiplicaion_11[3-33]" time="0.001" /><testcase classname="test_multiplication" name="test_mutiplicaion_11[4-44]" time="0.001" /></testsuite></testsuites>